common/processes/platform/tiny_process_library/tiny_process_library.cpp

```diff
@@ -17,30 +17,163 @@
 #include <string>
 #include <string_view>
 
+#include <common/events_io.hpp>
+#include <common/io.hpp>
+#include <common/log.hpp>
+
 using namespace std;
 
 namespace mender::common::processes {
 
+const chrono::seconds MAX_TERMINATION_TIME(10);
+
+namespace io = mender::common::io;
+namespace log = mender::common::log;
+
+class ProcessReaderFunctor {
```

Comment:
Hah, had me confused till I realised that a `functor` in Cpp is simply a function object.